### PR TITLE
[MRG] FIX regexp for KIT systems

### PR DESCRIPTION
**Pull request by jasmainak**
Looks like the regexp for KIT systems was still broken in the validator. I added some test cases (copied from the specification) now to make sure that it is indeed the right search pattern.

### CODE REVIEW

**Comment by jasmainak** [comment]
@chrisfilo : on reading the BIDS specification carefully, I see the following phrase for KIT/KRISS/BTi/ITAB systems:

> One   SHOULD   rename/create   a   father   run   specific   directory  and keep the original files for each run inside.

So, we cannot really check the extensions of the files or the files inside the father directory. Accordingly, I added more test cases and changed the regexp. It's good to go on my end.

**Comment by chrisgorgo** [comment]
@guiomar and @robertoostenveld know better this part of the spec

**Comment by jasmainak** [comment]
maybe @teonbrooks can help? He's a KIT systems expert :-)

File: tests/type.spec.js [hunk 1]
```diff
@@ -221,6 +225,8 @@ var suiteMEG = describe('utils.type.isMEG', function(){
         });
 
         var badFilenames = [
+            // only parent directory name matters for KIT/BTi systems
+            "/sub-01/ses-001/meg/sub-01_ses-001_task-rest_run-01_megggg/sub-01_ses-001_task-rest_run-01_meg.sqd",
```
Comment (teonbrooks) [inline tests/type.spec.js]: `_megggg` -> `_meg`

**Comment by guiomar** [comment]
Yes, we agreed that for systems that return more than one file (i.e. KIT, 4D/BTi, etc.) The best option was to create a father directory and include all of them inside. Sorry, I can't help more with this, I mainly work with Elekta and CTF.

There were also some comments (by Matt Sanderson) about homogenizing the folder structure for all the systems, creating always a father directory, even though the content inside will be sometimes just one file (.fif) or a folder with the exact same name (.ds), which may be a bit redundant, I don't have a strong opinion. However, if this helps for consistency, we may go for it.

**Pull request merged**